pkgs/development/ocaml-modules/middleware/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildDunePackage
+, fetchurl
+, alcotest
+}:
+
+buildDunePackage rec {
+  pname = "middleware";
+  version = "0.0.1";
+
+  minimalOCamlVersion = "4.14.0";
+
+  src = fetchurl {
+    url = "https://github.com/skolemlabs/middleware/releases/download/${version}/${pname}-${version}.tbz";
+    hash = "sha256-nkINT+Fe9pDUgS8/u8hhqA8MJdCsHbzxVewTyvv6F8k=";
```

Comment:
Are you sure about the hash?

Reply:
Good catch!

I guess the hash changed somehow -- I think I ended up re-publishing this release.

I've updated the hash. Should be good to go!